packages/integration-objects/srv/integration-objects/simple-tenant-configuration-for-conversion-service.js

```diff
@@ -85,21 +84,20 @@ module.exports = srv => {
       isConfigurationActive: data.isConfigurationActive
     };
 
-    let currentValuesIsNotEmpty = true;
-    if (Constants.CREATE_EVENT !== req.event) {
-      currentValues = await fetchCurrentAttributeValues(req, req.user.tenant, TenantConfigForConversions);
-      currentValuesIsNotEmpty = !util.isNullish(currentValues);
-    }
-
-    if (currentValuesIsNotEmpty) {
-      await writeAuditLog(req, auditParams, currentValues, type);
+    if (req.event === Constants.CREATE_EVENT) {
```

Comment:
???? This is the exact same code as in the other service. Please avoid code duplication and reuse existing functionality.